simplified-app-shared/src/main/java/org/nypl/simplified/app/catalog/CatalogFeedActivity.java

```diff
@@ -748,14 +734,33 @@ public Boolean onFeedSearchLocal(
             return NullCheck.notNull(Boolean.TRUE);
           }
         });
+
+    } else {
+      CatalogFeedActivity.LOG.debug("Feed has no search opts.");
     }
 
     if (search_ok) {
+      this.search_view.setSubmitButtonEnabled(true);
+      this.search_view.setIconifiedByDefault(false);
+      search_item.setShowAsActionFlags(MenuItem.SHOW_AS_ACTION_ALWAYS);
+      search_item.expandActionView();
+
+      // display either the category title or the previously searched keywords 
+      this.search_view.setQueryHint(String.format("%s %s", getString(R.string.search_hint_prefix), Objects.toString(this.feed.getFeedTitle(), "")));
```

Comment:
The token for the format string should itself be in the xml file, since not every language may structure the statement in the same way, so it'd be something like `String.format(R.string.search_hint,this.feed.getFeedTitle());` with the string being `"Search %s"` (in english).

I also believe that `getFeedTitle()` should be enough to create a String without the `Objects.toString()` call

Reply:
default case:
"Search [feed title]".  If feed title is null, then hint should be: "Search feed".  If feed title is not null, and already contains the word "Search", then hint should be "Search [feed title]".